### PR TITLE
Fixed logging when module is not loaded in otto or motto

### PR DESCRIPTION
**Pull request by wsikorska-codilime**
We need more logs to debug issues with loading modules in otto or motto. This PR just adds more logs.

### CODE REVIEW

File: extension/otto/gohan_core.go [hunk 1]
```diff
@@ -193,12 +195,15 @@ func requireFromOtto(moduleName string, vm *otto.Otto) (otto.Value, error) {
 }
 
 func requireFromMotto(moduleName string, vm *motto.Motto) (otto.Value, error) {
+	log.Info(fmt.Sprintf("Loading module %s from motto", moduleName))
```
Comment (vozhyk-) [inline extension/otto/gohan_core.go]: This can produce too many messages. What about `log.Debug`?

Comment (nati) [inline extension/otto/gohan_core.go]: I agree

**Comment by nati** [comment]
@wsikorska-codilime How about chaning level to debug?

**Comment by nati** [comment]
Could you push rebased version? CI should work now.

**Pull request merged**